qiskit/circuit/quantumcircuit.py

```diff
@@ -5199,3 +5211,22 @@ def _bit_argument_conversion_scalar(specifier, bit_sequence, bit_set, type_):
         else f"Invalid bit index: '{specifier}' of type '{type(specifier)}'"
     )
     raise CircuitError(message)
+
+
+def _instruction_qasm2(operation):
+    """Return an OpenQASM 2 string for the instruction."""
+    name_param = operation.name
+    if operation.params:
+        name_param = "{}({})".format(
+            name_param,
+            ",".join([pi_check(i, output="qasm", eps=1e-12) for i in operation.params]),
+        )
+    if operation.condition is not None:
+        if not isinstance(operation.condition[0], ClassicalRegister):
+            raise QasmError(
+                "OpenQASM 2 can only condition on registers, but got '{operation.condition[0]}'"
+            )
+        name_param = (
+            "if(%s==%d) " % (operation.condition[0].name, operation.condition[1]) + name_param
+        )
+    return name_param
```

Comment:
I get that it was pre-existing, but while we're at it, can we call `name_param` here (and all the places in `QuantumCircuit.qasm`) something that actually describes it - it's not just a name, it's a potentially conditional call with parameters.

Reply:
Maybe `qasm2_call` ?

Reply:
`qasm2_call` sounds like the best option - agree that it's not perfect, but at the very least it's not misleading like `name_param`.